website/content/gateway/api_reference/resources/debit_refunds/debit_refunds.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "debit_refunds"
+date: 2023-01-05T13:37:00+01:00
+anchor: "debit_refunds"
+weight: 178
+---
+#### Debit refunds
+To refund money to a cardholder's bank account that has been transferred by a debit you make a new refund resource. Only a single refund can be made for a debit and it can only be for the full amount.
+
+```shell
+POST https://gateway.clearhaus.com/debits/:id/refunds
+```
+
+##### Parameters
+No parameters are needed to make a new refund transaction.
+
+{{% notice %}}
+**Notice**: A refund of a Visa AFT debit might be declined by Visa or the issuer if the debit age is 24 hours or more, or if settlement has taken place for the debit.
```

Comment:
It has been pointed out that "settlement" here is imprecise.

Unfortunately, I have not been able to find any details in the Visa documentation I have read so far.

See https://clearhaus.slack.com/archives/C02V02D1SSC/p1672938054830459?thread_ts=1672935881.101459&cid=C02V02D1SSC.

Reply:
I have tried to clarify matters in https://github.com/clearhaus/gateway-api-docs/pull/146/commits/ad4f0b84db62fc8b84e58c799719a6c7d7572af3.